mteb/leaderboard/app.py

```diff
@@ -22,6 +23,13 @@
 from mteb.leaderboard.figures import performance_size_plot, radar_chart
 from mteb.leaderboard.table import scores_to_tables
 
+logging.getLogger("mteb.load_results.task_results").setLevel(
+    logging.WARNING
+)  # Warnings related to task split
+logging.getLogger("mteb.models.overview").setLevel(
+    logging.WARNING
+)  # Warning related to model metadata (fetch_from_hf=True)
```

Comment:
What happens if we set it to false in the leaderboard app?

Reply:
But could we then remove this filter?

Reply:
I have tried commenting this filter and running leaderboard, it is giving following warnings: 
<details>
<summary Warning: </summary>
<img width="1227" alt="Screenshot 2025-03-17 at 4 24 19 PM" src="https://github.com/user-attachments/assets/69984d72-4d3b-432b-8659-f0645db9acb4" />
</details>

Reply:
Ah, yes. This message will be createing https://github.com/embeddings-benchmark/mteb/blob/4c91ba9530306d4d16dcb3ed00d0ce54d2b2536d/mteb/model_meta.py#L189

Reply:
So, what should I do for this?

Reply:
I think you can leave it
```suggestion
)  # Warning related to model metadata (fetch_from_hf=False)
```

Reply:
Done